af/fastpath/fastpath/db.py

```diff
@@ -148,3 +156,141 @@ def upsert_summary(
         notification_json = ujson.dumps(notification)
         q = f"SELECT pg_notify('fastpath', '{notification_json}');"
         cur.execute(q)
+
+
+# # Clickhouse
+
+
+def _ch_ping():
+    result = _ch_send_query("POST", "SELECT version()")
+    log.info(result)
+
+
+def _ch_create_table_fastpath():
+    # _ch_send_query("POST", "DROP TABLE fastpath")
+    sql = """
+    CREATE TABLE IF NOT EXISTS fastpath (
+        "tid" String,
+        "report_id" String,
+        "input" String,
+        "probe_cc" String,
+        "probe_asn" Int32,
+        "test_name" String,
+        "test_start_time" DateTime,
+        "measurement_start_time" DateTime,
+        "platform" String,
+        "filename" String,
+        "scores" String,
+        "anomaly" Int8,
+        "confirmed" Int8,
+        "msm_failure" Int8,
+        "control_failure" String,
+        "is_ssl_expected" Int8,
+        "page_len" Int32,
+        "page_len_ratio" Float32,
+        "server_cc" String,
```

Comment:
It would also be useful to have the `blocking` determination done by the probe as a column on which to do feature extraction.

Other neat columns are:
`http_experiment_failure`
`dns_experiment_failure`